daemon/cmd/daemon_main.go

```diff
@@ -481,8 +481,9 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	flags.String(option.IPv6ServiceRange, AutoCIDR, "Kubernetes IPv6 services CIDR if not inside cluster prefix")
 	option.BindEnv(vp, option.IPv6ServiceRange)
 
-	flags.Bool(option.K8sEventHandover, defaults.K8sEventHandover, "Enable k8s event handover to kvstore for improved scalability")
```

Comment:
Where was this option marked as deprecated?

Reply:
It was marked as deprecated in the operator: https://github.com/cilium/cilium/blob/5ece06b065abf3660ebcea7d867c9be2fe44a01a/operator/cmd/flags.go#L62
unfortunately, I forgot to do it for the agent...
But we also mentioned it in the upgrade notes - https://docs.cilium.io/en/stable/operations/upgrade/#deprecated-options